charts/vector/docs/eks_quickstart.yaml

```diff
@@ -0,0 +1,21 @@
+## See Vector helm documentation to learn more:
+## https://vector.dev/docs/setup/installation/package-managers/helm/
+
+## Configuration for Vector's Service
+service:
+  # service.type -- Set type of Vector's Service
+  type: "LoadBalancer"
```

Comment:
As I understand it, this will _always_ create an NLB, and we'd need to use an `Ingress` to provision an ALB. From what I can tell (from [here](https://aws.amazon.com/elasticloadbalancing/features/)):

```
ALB can handle: HTTP/S, gRPC
NLB can handle: TCP, UDP, TLS
```

We likely need some more hands-on testing/knowledge on what we'd prefer and the limitations of each. The `eks_quickstart.yaml` could also be split into a `eks_alb|nlb_quickstart.yaml` as well.